table/updates.go

```diff
@@ -462,3 +468,35 @@ func NewRemoveSnapshotRefUpdate(ref string) *removeSnapshotRefUpdate {
 func (u *removeSnapshotRefUpdate) Apply(builder *MetadataBuilder) error {
 	return fmt.Errorf("%w: %s", iceberg.ErrNotImplemented, UpdateRemoveSnapshotRef)
 }
+
+type removeSpecUpdate struct {
+	baseUpdate
+	SpecIds []int64 `json:"spec-ids"`
+}
+
+func NewRemoveSpecUpdate(specIds []int64) Update {
```

Comment:
same as above, convention is to return the concrete type instead of the interface